diff -up corrosion-0.6.1/CMakeLists.txt.old corrosion-0.6.1/CMakeLists.txt
--- corrosion-0.6.1/CMakeLists.txt.old	2026-01-17 11:46:47.000000000 +0100
+++ corrosion-0.6.1/CMakeLists.txt	2026-06-16 16:01:01.523497114 +0200
@@ -77,5 +77,5 @@ install(
         cmake/CorrosionGenerator.cmake
         cmake/FindRust.cmake
     DESTINATION
-        "${CMAKE_INSTALL_FULL_DATADIR}/cmake"
+        "${CMAKE_INSTALL_FULL_DATADIR}/cmake/Modules"
 )
